Documentation/ProjectFile/prj/processes/process/SMALL_DEFORMATION/t_deactivated_subdomains.md

```diff
@@ -0,0 +1,3 @@
+Set of material subdomain indices. The elements in the subdomains by these
+indices will be skipped for local assembly.
```

Comment:
Set of material ids, which are excluded from local assembly.

Reply:
Changed to
```
Set of material indices, the subdomains by which are excluded from local assembly.
```

Reply:
"by which are" yields zero hits in google.